Payments/Operation/05_ManageConfirmations.md

```diff
@@ -0,0 +1,123 @@
+4. If required, check the details of the refund. For more information, see the following:
+     - [LIST &ndash; Attributes](../UserInterface/06_ListConfirmations.md#confirmations-–-attributes)
+
+
+
+## Execute confirmation transaction
+
+After you have checked a confirmation transaction and the reclaim is justified, you can execute it. The payment service provider can than arrange the repayment of the money.  
+
+#### Prerequisites
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one refund has been created.
+- At least the *Amount* and the *Status* column is displayed in the view.
+- The status of a confirmation transaction is "Confirmation required". 
+
+#### Procedure
+*Payments > Confirmations > Tab LIST*
+
+![Execute confirmation](../../Assets/Screenshots/Payments/Confirmations/ChangeConfirmation.png "[Execute confirmaton]")
+
+1. If desired, click the ![Search](../../Assets/Icons/Search.png "[Search]") (Search) button to display the search bar and search for the original reference ID of the refund.
+2. Check the confirmation transaction you want to execute, see [Check the confirmation transaction](05_ManageConfirmations.md#check-confirmation-transaction).
+2. Select the confirmation transaction you want to execute by clicking the checkbox on the left.   
+    The editing toolbar is displayed.
+3. Click the [Execute] button.
+   The entry disappears from the list. The workflow now initiates a repayment of the money.
+
+
+
+## Reject confirmation transaction
+
+After you have checked a confirmation transaction and the reclaim is **not** justified, you can reject it.   
+
+#### Prerequisites
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one refund has been created.
+- At least the *Amount* and the *Status* column is displayed in the view.
+- The status of a confirmation transaction is "Confirmation required". 
+
+#### Procedure
+
+*Payments > Confirmations > Tab LIST*
+
+![Reject confirmation](../../Assets/Screenshots/Payments/Confirmations/ChangeConfirmation.png "[Reject confirmaton]")
+
+
+1. If desired, click the ![Search](../../Assets/Icons/Search.png "[Search]") (Search) button to display the search bar and search for the original reference ID of the refund.
+2. Check the confirmation transaction you want to reject, see [Check the confirmation transaction](./05_ManageConfirmations.md#check-confirmation-transaction).
+2. Select the confirmation transaction you want to reject by clicking the checkbox on the left.   
+    The editing toolbar is displayed.
+3. Click the [Reject] button. 
+   A confirmation message is displayed.    
+   The status of the confirmation transaction will be internally changed to *Void*, but it will not be displayed. The workflow informs the payment service provider about the rejection of the refund.
```

Comment:
```suggestion
   The status of the confirmation transaction will be internally changed to **Void**, but it will not be displayed. The workflow informs the payment service provider about the rejection of the refund.
```

Reply:
ja